modules/installation-aws-config-yaml.adoc

```diff
@@ -343,14 +358,14 @@ ifdef::openshift-origin[]
 <12> How to publish the user-facing endpoints of your cluster. Set `publish` to `Internal` to deploy a private cluster, which cannot be accessed from the internet. The default value is `External`.
 endif::openshift-origin[]
 endif::private[]
-ifdef::gov[]
+ifdef::gov,secret[]
```

Comment:
The custom CA certificate is not required in the government partition.
```suggestion
ifdef::secret[]
```

Reply:
Done.